src/registrar/tests/test_admin.py

```diff
@@ -341,9 +341,9 @@ def test_get_filters(self):
             )
 
             # Assert that the filters are added
-            self.assertContains(response, "invited", count=5)
-            self.assertContains(response, "Invited", count=2)
-            self.assertContains(response, "retrieved", count=4)
+            self.assertContains(response, "invited", count=4)
```

Comment:
These and the other adjusted counts in this PR are changed because one of the libraries that got updated changed the way the response html shows up. Previous, a lot more cruft (including queries) were showing up. With the updated, there's a lot less included in the response.

The tests in general need some work. They are not very clear what data is created or what data should be expected. For now they are basically serving as a "snapshot", so if something changes, we'll know. In the future, we should adjust the tests to be more effective and focused.